backend/btrixcloud/crawls.py

```diff
@@ -1141,6 +1118,31 @@ async def get_qa_run_aggregate_stats(
             textMatch=text_results,
         )
 
+    async def get_crawl_logs(
+        self,
+        org: Organization,
+        crawl_id: str,
+        page_size: int = DEFAULT_PAGE_SIZE,
+        page: int = 1,
+        sort_by: str = "timestamp",
+        sort_direction: int = -1,
```

Comment:
```suggestion
        sort_direction: int = 1,
```
Use ascending timestamp sort order by default for consistency